src/libvfcfuncinstrument/libVFCFuncInstrument.cpp

```diff
@@ -396,58 +546,60 @@ struct VfclibFunc : public ModulePass {
             haveFloatingPointArithmetic(pi, f, is_from_library, is_intrinsic,
                                         &use_float, &use_double, M);
 
-            if (!(is_intrinsic && (!use_float && !use_double))) {
-              // Create function ID
-              Value *FunctionID = Builder.CreateGlobalStringPtr(FunctionName);
-
-              // Constants creation
-              Constant *isLibraryFunction = ConstantInt::get(
-                  Type::getInt8Ty(M.getContext()), is_from_library);
```

Comment:
This pattern `ConstantInt::get(Type::getInt8Ty(M.getContext()), ...` comes so often that the code would gain if we could make it simpler or factorize it.

It think LLVM IRBuilder has a much quicker way of getting constants. Any reason for not using it ?

Reply:
It's only because it's harder to get a "FloatPtrTy" and a "DoublePtrTy" with LLVM IRBuilder, but I can change as many calls as I can with LLVM Builder.

Reply:
I've made some changes, tell me if it's good for you?